cdhweb/pages/migrations/0004_homepage_hero_image_homepage_summary_and_more.py

```diff
@@ -0,0 +1,638 @@
+# Generated by Django 5.0.5 on 2024-05-08 22:33
+
+import django.db.models.deletion
+import wagtail.blocks
+import wagtail.documents.blocks
+import wagtail.embeds.blocks
+import wagtail.fields
+import wagtail.images.blocks
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
```

Comment:
It looks like this migration is adding the HomePageHeroMixin to the Homepage, but I can't see that change in this PR anywhere?

Reply:
sorry forgot to add it - there now